File: test7C#/test7/Model/Donut/SimpleDonut.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace test7.Model
{
    class SimpleDonut: CommandItem
    {
        private string flavor;

        public SimpleDonut(string name): base(name, Convert.ToDecimal(0.75), 1)
        {
            this.flavor = "plain";
        }

        public string Flavor() 
        {
            return flavor;
        }

        public void Flavor(string flavor)
        {
            this.flavor = flavor;
        }
    }
}

File: test7Go/oop/models/donut/simpleDonut.go
package donut

import (
	oop "oop/models"
)

type SimpleDonut struct {
	Flavor string
	oop.CommandItem
}

func NewSimpleDonut(name string) SimpleDonut {
	flavor := "plain"
	commandItem := oop.CommandItem{Name: name, Price: 0.75, Quantity: 1}

	return SimpleDonut{flavor, commandItem}
}
